server/src/logic/reporter/report.js

```diff
@@ -10,7 +10,7 @@ const getEmailTemplate = (type = "report") => {
 };
 
 const generate = async (data, title, to, templateName = "report") => {
-  const date = DateTime.local().setLocale("fr").toFormat("dd MMMM yyyy");
+  const date = DateTime.local().plus({ days: 1 }).setLocale("fr").toFormat("dd MMMM yyyy");
```

Comment:
Est-ce qu'on peut tester en changeant la timezone de luxon ?
```suggestion
  const date = DateTime.local({ zone: "Europe/Paris" }).setLocale("fr").toFormat("dd MMMM yyyy");

```